test_battleship.py

```diff
@@ -135,4 +232,19 @@ def test_start_game(self):
         observed_primary_board_player_one = self.battleship.get_primary_board_player_one()
 
         # assert
-        self.assertEqual(expected_primary_board_player_one, observed_primary_board_player_one)
\ No newline at end of file
+        self.assertEqual(expected_primary_board_player_one, observed_primary_board_player_one)
+
+
+    # def test_player_pick_points(self):
```

Comment:
best not to commit comments into version control. 
reason is because you can look back at git history to find what you've commented out.

Reply:
Resolved.